--- dist/keyringDbus.js	2022-11-14 14:04:06.268303707 +0100
+++ dist/keyringDbus.patched.js	2022-11-14 14:04:02.681319328 +0100
@@ -32,12 +32,6 @@
             Pango: { id: 'Pango', exports: imports.gi.Pango },
             GObject: { id: 'GObject', exports: imports.gi.GObject }
         })) },
-    console: { value: {
-            log: function () { print.apply(null, arguments); },
-            warn: function () { print.apply(null, arguments); },
-            error: function () { print.apply(null, arguments); },
-            info: function () { print.apply(null, arguments); }
-        } },
     _: { value: imports.gettext.gettext }
 });
 Object.defineProperties(define, {
